python/lsst/faro/utils/selectors.py

```diff
@@ -518,10 +519,14 @@ def brightIsolatedStarObjectTable(config):
     # setup non band flag slectors
     config.selectorActions.FlagSelector = FlagSelector
     config.selectorActions.FlagSelector.selectWhenTrue = ["detect_isPrimary"]
+    config.selectorActions.FlagSelector.selectWhenFalse = ["xy_flag"]
     config.selectorActions.FlagSelector.selectorBandType = "currentBands"
     # setup per band flag selectors
     config.selectorActions.PerBandFlagSelector = PerBandFlagSelector
-    config.selectorActions.PerBandFlagSelector.selectWhenFalse = ["pixelFlags_saturated", "pixelFlags_cr",
-                                                                  "pixelFlags_bad", "pixelFlags_edge"]
+    # config.selectorActions.PerBandFlagSelector.selectWhenFalse = ["pixelFlags_saturated", "pixelFlags_cr",
+    #                                                               "pixelFlags_bad", "pixelFlags_edge"]
+    config.selectorActions.PerBandFlagSelector.selectWhenFalse = ["psfFlux_flag",
+                                                                  "pixelFlags_saturatedCenter",
+                                                                  "extendedness_flag"]
     config.selectorActions.PerBandFlagSelector.selectorBandType = "currentBands"
```

Comment:
I think instead of all of this it should use the predefined coadd selector that is used for all of the plots, it is less complicated and means that we have one defined set of flags that we consider the bases ones for "good qa sources". I think it is very similar to what you have here but easier to keep updated and consistent between metrics or plots. You can add separate ones on top when you require additional flags, I'm not actually sure why you would need a PerBandFlagSelector when FlagSelector (at least the original) takes bands as a config option.

Reply:
Also commented out lines again.

Reply:
> I think instead of all of this it should use the predefined coadd selector that is used for all of the plots, it is less complicated and means that we have one defined set of flags that we consider the bases ones for "good qa sources". I think it is very similar to what you have here but easier to keep updated and consistent between metrics or plots. You can add separate ones on top when you require additional flags, 

That is exactly what the "brightIsolatedStarObjectTable" selector does if left to its defaults -- apply the same quality cuts that are used in "CoaddPlotFlagSelector" from analysis_drp. The way "CoaddPlotFlagSelector" is written does not work for use with `faro` because there is flexibility we needed that didn't seem possible with some of the analysis_drp coding choices. I completely agree that we should ensure consistency (where possible) in all analyses. That is why we included "brightIsolatedStarObjectTable" with the "standard good QA sources" selection.

> I'm not actually sure why you would need a PerBandFlagSelector when FlagSelector (at least the original) takes bands as a config option.

This is because some flags are defined "per band" (i.e., there is a separate flag for each band, so the band name needs to be prepended to the flag name) and some are defined once in the entire table (e.g., flags in `SourceTable`s, which contain only a single band).

Reply:
But the selectors can use both per band flags and not per band flags otherwise we would have issues in analysis_drp. We should share the same selectors between the two.